staramr/blast/results/pointfinder/codon/CodonMutationPosition.py

```diff
@@ -80,10 +80,16 @@ def get_mutation_string(self):
                + ' -> ' + self.get_input_genome_amino_acid() + ')'
 
     def get_database_amr_gene_mutation(self):
-        return self.get_database_amr_gene_amino_acid().upper()
+        if '-' in self.get_database_amr_gene_codon():
```

Comment:
I think `upper()` will not modify `-`, so this could likely remain as:

```python
return self.get_database_amr_gene_amino_acid().upper()
```

Reply:
This isn't modifying the `-`, it's making sure `ins` and `del` are returned lowercase to match the Pointfinder resistens table entries. I realize that this isn't too clear in the code, so I'll make a comment about it.

Reply:
Oh, okay. Now I get it. That sounds good. Thanks.